includes.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Assert if a given array contains a value
+ * @module 101/includes
+ */
+'use strict';
+
+var isNumber = require('./is-number');
+
+/**
+ * @param {Array} array
+ * @param {*} searchElement
+ * @param {Number} fromIndex
+ * @return Boolean
+ */
+module.exports = function (array, searchElement, fromIndex) {
+  if (arguments.length === 1) {
+    return includes.bind(null, array);
+  } else {
+    return includes(array, searchElement, fromIndex);
+  }
+};
+
+function includes (array, searchElement, fromIndex) {
+  if (!isNumber(fromIndex)) {
+    fromIndex = 0;
+  }
+  if (Array.prototype.includes) {
+    return array.includes(searchElement, fromIndex);
+  } else {
+    // ES7 Array.prototype.includes polyfill (modified)
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility
+    var O = Object(array);
+    var len = parseInt(O.length) || 0;
+    if (len === 0) {
+      return false;
+    }
+    var n = fromIndex;
+    var k;
+    if (n >= 0) {
+      k = n;
+    } else {
+      k = len + n;
+      if (k < 0) {k = 0;}
+    }
+    var currentElement;
+    while (k < len) {
+      currentElement = O[k];
+      if (searchElement === currentElement ||
+         (searchElement !== searchElement && currentElement !== currentElement)) {
```

Comment:
Some more information here: https://tc39.github.io/Array.prototype.includes/
`If SameValueZero(searchElement, elementK) is true, return true.`